.test-defs/TestSuiteShootRsyslogRelpBetaSerial.yaml

```
@@ -21,5 +21,6 @@ spec:
       -shoot-name=$SHOOT_NAME
       -ginkgo.focus="\[BETA\].*\[SERIAL\]"
       -ginkgo.skip="\[DISRUPTIVE\]"
+      -ginkgo.timeout=2h
```

I would align this timeout with `activeDeadlineSeconds` as we were maintaining in activeDeadlineSeconds the sum of all test timeouts.

```suggestion
      -ginkgo.timeout=12000s
```

We have 2 tests using 50min as timeout, and 1 test now using 75min, so these 2h might not be enough in all cases.

---

Yeah, let's keep it consistent. Actually 13200 should be the correct value for `activeDeadlineSeconds`. I guess 12300 was a typo or miscalculation. I've also set the `ginkgo.timeout` to that value as well.